src/resources/Organizations/Organization.ts

```diff
@@ -18,14 +24,18 @@ export default class Organization extends Resource {
         return this.api.delete(`${Organization.baseUrl}/${organizationId}`);
     }
 
-    get(organizationId: string) {
-        return this.api.get<OrganizationModel>(`${Organization.baseUrl}/${organizationId}`);
+    get(organizationId: string, options?: GetOrganizationOptions) {
+        return this.api.get<OrganizationModel>(this.buildPath(`${Organization.baseUrl}/${organizationId}`, options));
     }
 
     update(organization: Partial<OrganizationModel>) {
         return this.api.put<OrganizationModel>(`${Organization.baseUrl}/${organization.id}`, organization);
     }
 
+    status(organizationId: string = API.orgPlaceholder) {
+        return this.api.get<OrganizationsStatusModel>(`${Organization.baseUrl}/${organizationId}/status`);
+    }
+
```

Comment:
I'm not sure which is better 🤔 
`status() {
       return this.api.get<OrganizationsStatusModel>(`${Organization.baseUrl}/${this.api.organizationId}/status`);
}`

Reply:
I would rather keep the optional parameter, it defaults to `API.orgPlaceholder` anyway